adafruit_turtle.py

```diff
@@ -115,7 +95,7 @@ class Vec2D(tuple):
     #     |a| absolute value of a
     #     a.rotate(angle) rotation
     def __init__(self, x, y):
-        super(Vec2D, self).__init__((x, y))
+        super().__init__((x, y))
```

Comment:
Just wanted to verify that this was done correctly. I seem to remember doing it this way before, but I'm not entirely sure

Reply:
I've never written a super with the class and `self` in it, so I'm assuming this is correct.

Reply:
Ok. Good to know. There are a few other PRs where I left the same comment